### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by rschik**
Replaced constant 24 with phrases.length-1 in the re-try path

### CODE REVIEW

File: js/app.js
Comment (Warnstrom) [inline js/app.js:1]: Could we assign the phrases.length-1 to an variable to avoid .length being called each time?

File: js/app.js [hunk 1]
```diff
@@ -274,7 +274,7 @@ const generateBingoCards = async () => {
   for (let i = 0; i < 25; ) {
     let n = randomNumber(0, phrases.length-1);
     if (usedNumbers.includes(n)) {
-      n = randomNumber(0, 24);
+      n = randomNumber(0, phrases.length-1);
```
Comment (Warnstrom) [inline js/app.js]: Consider assigning phrases.length - 1 to a variable to avoid redundant calls to .length.

Comment (rschik) [reply js/app.js]: Sounds like a good idea! Will add that later.

Comment (rschik) [reply js/app.js]: Added a variable for that

**Pull request merged**